public/scripts/slash-commands.js

```diff
@@ -354,8 +354,8 @@ function trimTokensCallback(arg, value) {
         }
 
         const sliceTokens = direction === 'start' ? textTokens.slice(0, limit) : textTokens.slice(-limit);
-        const decodedText = decodeTextTokens(tokenizerId, sliceTokens);
-        return decodedText;
+        const { text } = decodeTextTokens(tokenizerId, sliceTokens);
+        return text;
```

Comment:
I had to make a breaking change to the signature of `decodeTextTokens` to support returning both a full string and the individual tokens, but fortunately this /trimtokens slash command handler is the only thing that was using it,